<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find snap_circuits)/rosconsole.conf"/>

  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="960" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="board_calibrator" pkg="snap_circuits" type="board_calibrator" respawn="false" output="screen">
    <param name="show" value="true" />
    <param name="sub"  value="/usb_cam/image_raw" />
    <param name="pub"  value="/board_calibrator/image_undistorted" />
  </node>

  <node name="image_undistorted" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/board_calibrator/image_undistorted"/>
    <param name="autosize" value="true" />
  </node>

</launch>
